packages/replay/test/unit/index-errorSampleRate.test.ts

```diff
@@ -60,9 +60,9 @@ describe('Replay (errorSampleRate)', () => {
     expect(replay).toHaveSentReplay({
       recordingPayloadHeader: { segment_id: 0 },
       replayEventPayload: expect.objectContaining({
+        replay_type: 'error',
```

Comment:
Just as a side note: I'm not too happy about assertions like `expect.objectContaining` because they don't catch certain kinds of changes to the object under test. In this example, when we add a field, we need to specifically look up these tests and add them to cover the new field. Also, if we "accidentally" add a field, this assertion wouldn't catch it. If we just use "hard" comparisons, I think we could increase the robustness of these tests, as they would fail automatically. It's not something we have to change right now but I think, we should keep it in mind going forward.